provider/cpu_utils.py

```diff
@@ -98,8 +100,18 @@ def check_if_vm_vcpu_topology_match(session, os_type, cpuinfo):
                               " returned by guest:\n%s", out)
                 return False
 
+    threads_matched = threads//cores == cpuinfo.threads
+    if devices.qemu_version in VersionInterval("[, 8.0.0)"):
```

Comment:
IMO, it is `(,8.1.0)` and it is for windows guest only.

Reply:
Yes, correct, I was modifying the logic of my code a bit,  I am reverting this part now.

Reply:
It is not related with the start or `(` or `[`.   It is about the end, the difference with `8.0.0` and `8.1.0`.    Use `8.1.0)` is must IMO.  Any `8.1.0-***` will be skipped.